tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java

```diff
@@ -237,7 +237,7 @@ public void basicRenameTest7() throws Exception {
     // Due to Hadoop 1 support we stick with the deprecated version. If we drop support for it
     // FSDataOutputStream.hflush will be the new one.
     //#ifdef HADOOP1
-    o.sync();
+//    o.sync();
```

Comment:
Is this an accident removed?

Reply:
adding it back, sorry for the inconvenience (or you are welcome for the convenience 😂) https://github.com/Alluxio/alluxio/pull/15787